# Polyploid



- #### [The Evolution of Chromosome Numbers: Mechanistic Models and Experimental Approaches](https://academic.oup.com/gbe/article/13/2/evaa220/5923296)



> Chromosome numbers have been widely used to describe the most fundamental genomic attribute of an organism or a lineage. Although providing strong phylogenetic signal, chromosome numbers vary remarkably among eukaryotes at all levels of taxonomic resolution. Changes in chromosome numbers regularly serve as indication of major genomic events, most notably polyploidy and dysploidy. Here, we review recent advancements in our ability to make inferences regarding historical events that led to alterations in the number of chromosomes of a lineage. We first describe the mechanistic processes underlying changes in chromosome numbers, focusing on structural chromosomal rearrangements. Then, we focus on experimental procedures, encompassing comparative cytogenomics and genomics approaches, and on computational methodologies that are based on explicit models of chromosome-number evolution. Together, these tools offer valuable predictions regarding historical events that have changed chromosome numbers and genome structures, as well as their phylogenetic and temporal placements.



```python


FURTHER READING
Tettelin, H. et al. Genome analysis of multiple pathogenic isolates of Streptococcus agalactiae: implications for the microbial “pan-genome”. Proc. Natl Acad. Sci. USA 102, 13950–13955 (2005).

Li, R. et al. Building the sequence map of the human pan-genome. Nat. Biotechnol. 28, 57–63 (2010).

Golicz, A. A. et al. The pangenome of an agronomically important crop plant Brassica oleracea. Nat. Commun. 7, 13390 (2016).

Montenegro, J. D. et al. The pangenome of hexaploid bread wheat. Plant J. 90, 1007–1013 (2017).

Zhao, Q. et al. Pan-genome analysis highlights the extent of genomic variation in cultivated and wild rice. Nat. Genet. 50, 278–284 (2018).

Gao, L. et al. The tomato pan-genome uncovers new genes and a rare allele regulating fruit flavor. Nat. Genet. 51, 1044–1051 (2019).

Alonge, M. et al. Major impacts of widespread structural variation on gene expression and crop improvement in tomato. Cell. 182, 145–161 (2020).

Liu, Y. et al. Pan-genome of wild and cultivated soybeans. Cell 182, 162–176 (2020).




```


- #### [The relation of meiotic behaviour to hybridity, polyploidy and apomixis in the Ranunculus auricomus complex (Ranunculaceae)](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-020-02654-3)

- #### [Polyploidy and high environmental tolerance increase the invasive success of plants](https://link.springer.com/article/10.1007/s10265-020-01236-6)


- #### [Subgenome evolution in allotetraploid plants](https://onlinelibrary.wiley.com/doi/pdf/10.1111/tpj.15190)


> Polyploidization is a well‐known speciation and adaptation mechanism. Traces of former polyploidization events were discovered within many genomes and especially in plants. Allopolyploidization by interspecific hybridization between two species is common. Among hybrid plants, many are domesticated species of agricultural interest and many of their genomes and of their presumptive parents have been sequenced. 


> Hybrid genomes remain challenging to analyse due to the presence of multiple subgenomes. The genomes of hybrids often undergo rearrangement and degradation over time. Based on ten hybrid plant genomes from six different genera with hybridization dating from 10,000 years to five million years ago, we assessed subgenome degradation, subgenomic intermixing, and biased subgenome fractionation. Restructuring of hybrid genomes does not proceed proportional to hybrid age. The oldest hybrids in our dataset displayed completely different fates: while the subgenomes of the tobacco Nicotiana benthamiana are in an advanced stage of degradation, those of quinoa (Chenopodium quinoa) are exceptionally well conserved by structure and sequence. We observed statistically significant biased subgenome fractionation in seven out of ten hybrids, which had different ages and subgenomic intermixing levels. 


> Hence, we conclude that no correlation exists between biased fractionation and subgenome intermixing. Lastly, domestication may both encourage or hinder subgenome intermixing depending on the evolutionary context. In summary, comparative analysis of hybrid genomes and their presumptive parents allowed us to determine commonalities and differences between their evolutionary fates. 


> Facilitating future analysis of further hybrid genomes, we automated the analysis steps within the program “Manticore”, publicly available at https://github.com/MatteoSchiavinato/manticore.git.
